ProjectLighthouse/Database/DatabaseContext.Notifications.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+using LBPUnion.ProjectLighthouse.Configuration;
+using LBPUnion.ProjectLighthouse.Types.Entities.Notifications;
+using LBPUnion.ProjectLighthouse.Types.Notifications;
+
+namespace LBPUnion.ProjectLighthouse.Database;
+
+public partial class DatabaseContext
+{
+    public async Task SendNotification
+        (int userId, string text, NotificationType type = NotificationType.ModerationNotification)
+    {
+        // Append server name to notification text if enabled
+        if (ServerConfiguration.Instance.NotificationConfiguration.ShowServerNameInText)
+        {
+            text = $"[{ServerConfiguration.Instance.Customization.ServerName}] {text}";
+        }
```

Comment:
Notifications should probably be configurable/changed to have a timestamp so that you can have a timestamp in the message.